documentation/dev-portal/src/nymvpn/cli-linux.md

```diff
@@ -56,16 +51,16 @@ In case of errors, see [troubleshooting section](troubleshooting.md).
 
 The basic syntax of `nym-vpn-cli` is:
 ```sh
-sudo ./nym-vpn-cli -c ./sandbox.env --entry-gateway-id <ENTRY_GATEWAY_ID> --exit-router-address <EXIT_ROUTER_ADDRESS> --enable-wireguard --private-key <PRIVATE_KEY> --wg-ip <WG_IP>
+sudo ./nym-vpn-cli <--exit-router-address <EXIT_ROUTER_ADDRESS>|--exit-gateway-id <EXIT_GATEWAY_ID>|--exit-gateway-country <EXIT_GATEWAY_COUNTRY>>
```

Comment:
what's going on with the first `<--` here? I'm a little confused by it

Reply:
are these optional values?

Reply:
Those are three option for one conditional arg, changed it.